templates/patterns/text_featured_media/pattern-text-featured-media.html.twig

```diff
@@ -2,23 +2,53 @@
 /**
  * @file
  * Text with featured media.
+ *
+ * Variant "left_simple" is used by default.
  */
 #}
 
+{% set variant = variant|default('left_simple') %}
+
 {% set video_ratio = video_ratio|replace({':': '-'}) %}
 {% set allowed_values = ['16-9','4-3','3-2','1-1'] %}
 {% if video_ratio not in allowed_values %}
   {% set video_ratio = '16-9' %}
 {% endif %}
 {% set text = text|render %}
 
-{% if image or video or text %}
+{% set media_position_class = ' ecl-u-order-md-last' %}
+{% if 'right' in variant %}
+  {% set media_position_class = '' %}
+{% endif %}
+
+{% set call_to_action_link = '' %}
+{% if link %}
+  {% set link_type = 'standalone' %}
```

Comment:
Same here, we could do {% set link_type = 'featured' in variant ? 'cta' : 'standalone' %}